lms/djangoapps/teams/views.py

```diff
@@ -1087,3 +1116,38 @@ def delete(self, request, team_id, username):
             return Response(status=status.HTTP_204_NO_CONTENT)
         else:
             return Response(status=status.HTTP_404_NOT_FOUND)
+
+
+class SearchResultPaginator(object):
```

Comment:
I had a question about where this logic belongs.  Do we break it out into its own file? Should this actually be a private method within `TeamsListView`? Is it fine where it is and we could refactor it if we need to use it elsewhere?